backend/src/main/java/gov/cdc/usds/simplereport/api/patient/PatientResolver.java

```diff
@@ -25,31 +26,42 @@ public PatientResolver(PersonService ps, OrganizationService os) {
     _os = os;
   }
 
+  private ArchivedStatus getArchivedStatus(Boolean includeArchived, ArchivedStatus archivedStatus) {
```

Comment:
⚠️ for backward compatibility -- will remove in separate PR once this one is deployed